wger/core/models.py

```diff
@@ -303,6 +303,12 @@ class UserProfile(models.Model):
                                     default=False)
     '''Allow anonymous read-only access'''
 
+    ''' Number of Days Reminder Weight '''
```

Comment:
Move this to *after* num_days, this has a special meaning to python and if we ever have automatic documentaion generation, these comments are automatically used.